doc/source/parameters/dem/dem.rst

```diff
@@ -1,3 +1,17 @@
 ***********************************************
 Discrete Element Method (DEM)
-***********************************************
\ No newline at end of file
+***********************************************
+
+
```

Comment:
I would add a few lines here that explain the core idea with the DEM parameters.

Reply:
Added.